components/user_settings/notifications/email_notification_setting/email_notification_setting.test.tsx

```diff
@@ -184,8 +185,8 @@ describe('components/user_settings/notifications/EmailNotificationSetting', () =
         expect(wrapper.state('sendEmailNotifications')).toBe(nextProps.sendEmailNotifications);
         expect(wrapper.state('newInterval')).toBe(Preferences.INTERVAL_NEVER);
 
-        nextProps.enableEmail = true;
         nextProps.emailInterval = Preferences.INTERVAL_FIFTEEN_MINUTES;
+        nextProps.enableEmail = true;
         wrapper.setProps(nextProps);
```

Comment:
Nit: Here you have change the order of the lines. Not a real problem, but if you can put it in the original order, it will make the changes log clearer.